tests/medcat/2_train_model/1_unsupervised_training/test_splitter.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+
+import tempfile
+import os
+import sys
+import pandas as pd
+
+_FILE_DIR = os.path.dirname(__file__)
+
+# because this project isn't (at least of of writing this)
+# set up as a python project, there are no __init__.py
+# files in each folder
+# as such, in order to gain access to the relevant module,
+# I'll need to add the path manually
+_WWC_BASE_FOLDER = os.path.join(_FILE_DIR, "..", "..", "..", "..")
+MEDCAT_EVAL_MCT_EXPORT_FOLDER = os.path.abspath(os.path.join(_WWC_BASE_FOLDER, "medcat", "2_train_model", "1_unsupervised_training"))
```

Comment:
Yeah, it's always a hack to do something like this.

I agree, we would probably be better of restructuring this in a different way.
If there are things in there that we deem shouldn't be in MedCAT (or another project), they should be handled in a better way here such that these types of hacks aren't necessary.
As to what a good solution to this would be is probably outside the scope of this PR.